README.md

```diff
@@ -50,6 +50,7 @@ sway-applications/
 - [Escrow](./escrow) is a third party that keeps an asset on behalf of multiple parties
 - [Fundraiser](./fundraiser/) is a program allowing users to pledge towards a goal
 - [Multi-Signature Wallet](./multisig-wallet) is a wallet that requires multiple signatures to execute a transaction
+- [English Auction](./auctions/english-auction/) is a English Auction template where user bid up the price of an asset until time is up or a reserve has been met.
```

Comment:
Move to top, check grammar for sentence and nest under

- Auctions
    - Auction 1
    - 2
    - 3
    - ...